public/examples/turtletest.md

```diff
@@ -0,0 +1,6 @@
elVuD569ZH0GoOe1KYsBPzk3v4GDHr7io59oEE/tjoPJ7AmYEDWnHZ9Sh+7pOd9j0Gf17nEwwQMSK/C9dGrj6TXGPSkNmUh4Cf39jdg0NtXdOwDDfqx1Xk4gTUBA7LmtOtT+tglPe97DPq8ziUeJmBAehWuj159JL3GoCe1KQsBP7m3vwGD3r6iYx9o0I+tzsMJrAkYkDWnXZ/Sxy7ped9j0Od1LvEwAQPSq3B99Ooj6TUGPalNWQj4yb39DRj09hUd+0CDfmx1Hk5gTcCArDnt+pQ+dknP+x6DPq9ziYcJGJBeheujVx9JrzHoSW3KQsBP7u1vwKC3r+jYBxr0Y6vzcAJrAgZkzWnXp/SxS3re9xj0eZ1LPEzAgPQqXB+9+kh6jUFPalMWAn5yb38DBr19Rcc+0KAfW52HE1gTMCBrTrs+pY9d0vO+x6DP61ziYQIGpFfh+ujVR9JrDHpSm7IQ8JN7+xsw6O0rOvaBBv3Y6jycwJqAAVlz2vUpfeySnvc9Bn1e5xIPEzAgvQrXR68+kl5j0JPalIWAn9zb34BBb1/RsQ806MdW5+EE1gQMyJrTrk/pY5f0vO8x6PM6l3iYgAHpVbg+evWR9BqDntSmLAT85N7+Bgx6+4qOfaBBP7Y6DyewJmBA1px2fUofu6TnfY9Bn9e5xMMEDEivwvXRq4+k1xj0pDZlIeAn9/Y3YNDbV3TsAw36sdV5OIE1AQOy5rTrU/rYJT3vewz6vM4lHiZgQHoVro9efSS9xqAntSkLAT+5t78Bg96+omMfaNCPrc7DCawJGJA1p12f0scu6XnfY9DndS7xMAED0qtwffTqI+k1Bj2pTVkI+Mm9/Q0Y9PYVHftAg35sdR5OYE3AgKw57fqUPnZJz/segz6vc4mHCRiQXoXro1cfSa8x6EltykLAT+7tb8Cgt6/o2Aca9GOr83ACawIGZM1p16f0sUt63vcY9HmdSzxMwID0KlwfvfpIeo1BT2pTFgJ+cm9/Awa9fUXHPtCgH1udhxNYEzAga067PqWPXdLzvsegz+tc4mECBqRX4fro1UfSawx6UpuyEPCTe/sbMOjtKzr2gQb92Oo8nMCagAFZc9r1KX3skp73PQZ9XucSDxMwIL0K10evPpJeY9CT2pSFgJ/c29+AQW9f0bEPNOjHVufhBNYEDMia065P6WOX9LzvMejzOpd4mIAB6VW4Pnr1kfQag57UpiwE/OTe/gYMevuKjn2gQT+2Og8nsCZgQNacdn1KH7uk532PQZ/XucTDBAxIr8L10auPpNcY9KQ2ZSHgJ/f2N2DQ21d07AMN+rHVeTiBNQEDsua061P62CU973sM+rzOJR4m8N8BeaOq7g6L3jXugy4uVe92faB3nSlg0M/szasJPJPAzV8VPtO/yYdfmIAxf2G0o//EBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAgY9JQm5SBAgACB0QIGfXT9whMgQIBAioBBT2lSDgIECBAYLWDQR9cvPAECBAikCBj0lCblIECAAIHRAgZ9dP3CEyBAgECKgEFPaVIOAgQIEBgtYNBH1y88AQIECKQIGPSUJuUgQIAAgdECBn10/cITIECAQIqAQU9pUg4CBAgQGC1g0EfXLzwBAgQIpAj8B30pAa9c9vkzAAAAAElFTkSuQmCC)
```

Comment:
as discussed, it would be nice to generate this in a special tag (as many challenges might want to have the target output inlined)